.travis.yml

```diff
@@ -31,13 +31,30 @@ install:
 before_script: '[ "$MAIN_STAGE" = true ] && greenkeeper-lockfile-update || true'
 script:
   - npm run build
+  - rake docs
   - '[ "$MAIN_STAGE" = true ] && ./scripts/check-pristine-state package-lock.json || true'
   - (cd Example && bundle exec rake ci)
 after_script: '[ "$MAIN_STAGE" = true ] && greenkeeper-lockfile-upload || true'
 branches:
   only:
     - master
     - /^greenkeeper/.*$/
+
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $IOS_DOCS_DEPLOY_TOKEN
+  # on:
+  #  branch: master
+  # TODO CHECK FOR TAGS
+  local_dir: docs/
```

Comment:
Can we make this `versions/{latest_version}`? So that we keep the history and can point to old releases?